packages/headless/src/controllers/core/context/headless-core-context.ts

```diff
@@ -11,13 +12,28 @@ import {
 } from '../../../features/context/context-state';
 import {ContextSection} from '../../../state/state-sections';
 import {loadReducerError} from '../../../utils/errors';
+import {validateInitialState} from '../../../utils/validate-payload';
 import {
   buildController,
   Controller,
 } from '../../controller/headless-controller';
 
 export type {ContextPayload, ContextValue};
 
+export interface ContextProps {
+  initialState?: ContextInitialState;
+}
+
+export interface ContextInitialState {
+  values: ContextPayload;
```

Comment:
`values` is pretty generic for a name :D 

But then, I dont have any good suggestion for an alternative name, so 🤷